react/src/components/ContainerRegistryEditorModal.tsx

```diff
@@ -118,36 +152,61 @@ const ContainerRegistryEditorModal: React.FC<
               : undefined, // no change
         };
         if (containerRegistry) {
-          if (!values.isChangedPassword) {
-            delete mutationVariables.password;
-          }
           mutationVariables = _.omitBy(mutationVariables, _.isNil) as Required<
             typeof mutationVariables
           >;
```

Comment:
We don't need this line. Relay sends a value as null when the mutation operation value of the commit function argument is null or undefined.